### PR TITLE
[org] Add org-appear support

### PR DESCRIPTION
**Pull request by winsphinx**
fixes for PR #14465

### CODE REVIEW

**Review by smile13241324 — APPROVED** [review]
Looks good will be merged soon.

File: layers/+emacs/org/packages.el [hunk 1]
```diff
@@ -50,7 +50,9 @@
     (org-sticky-header :toggle org-enable-sticky-header)
     (verb :toggle org-enable-verb-support)
     (org-roam :toggle org-enable-roam-support)
-    (valign :toggle org-enable-valign)))
+    (valign :toggle org-enable-valign)
+    (org-appear :location (recipe :fetcher github :repo "awth13/org-appear")
```
Comment (tko) [inline layers/+emacs/org/packages.el:54]: [org-appear seems to be in MELPA](https://melpa.org/#/org-appear) -- I wonder whether it or github is preferred from where to fetch the packages?

Comment (Kazark) [reply layers/+emacs/org/packages.el:54]: Good question.

Comment (winsphinx) [reply layers/+emacs/org/packages.el:54]: If `location` is set, github repo is preferred. So I think the one on melpa should be preferred.

**Pull request merged**